rust/src/measurements/make_private_expr/expr_index_candidates/mod.rs

```diff
@@ -42,8 +42,14 @@ where
     Expr: PrivateExpr<MI, MO>,
     (ExprDomain, MI): MetricSpace,
 {
-    let (input, IndexCandidatesArgs { candidates }) = match_index_candidates(&expr)?
-        .ok_or_else(|| err!(MakeMeasurement, "Expected Index Candidates function"))?;
+    let (input, IndexCandidatesArgs { candidates }) =
```

Comment:
Did you intend for this index_candidates code to be in this PR? I'm wondering if it got included by mistake...